`starts_on` + `let_value` returning `exec::task` fails to compile with `_NO_USABLE_CONNECT_CUSTOMIZATION_FOUND`

Calling `stdexec::starts_on(inline_scheduler, just() | let_value(...))` where the lambda passed to `let_value` returns `exec::task<void>` fails to compile.

### Minimal repro

```c++
#include <exec/task.hpp>
#include <stdexec/execution.hpp>

int main() {
  stdexec::starts_on(
      stdexec::inline_scheduler{},
      stdexec::just() |
          stdexec::let_value([]() -> exec::task<void> { co_return; }));
  return 0;
}
```

### Full error log

```
In file included from /Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__execution_fwd.hpp:19,
                 from /Users/kyusic/dev/stdexec/include/exec/../stdexec/execution.hpp:18,
                 from /Users/kyusic/dev/stdexec/include/exec/task.hpp:23,
                 from /Users/kyusic/dev/stdexec/test_exec.cpp:1:
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__meta.hpp: In substitution of 'template<class _Fun, class ... _As> using stdexec::__call_result_t = decltype (__declval<_Fun>()((__declval<_As>)()...)) [with _Fun = stdexec::__connect::connect_t; _As = {exec::__task::basic_task<void, exec::__task::__default_task_context_impl<exec::__task::__scheduler_affinity::__sticky> >, stdexec::__any_::__receiver_ref<stdexec::completion_signatures<stdexec::__rcvrs::set_value_t(), stdexec::__rcvrs::set_error_t(std::__exception_ptr::exception_ptr), stdexec::__rcvrs::set_stopped_t()>, stdexec::__env::env<> >}]':
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__senders_core.hpp:70:9:   required by substitution of 'template<class _Sender, class _Receiver> using stdexec::connect_result_t = stdexec::__call_result_t<stdexec::__connect::connect_t, _Sender, _Receiver> [with _Sender = exec::__task::basic_task<void, exec::__task::__default_task_context_impl<exec::__task::__scheduler_affinity::__sticky> >; _Receiver = stdexec::__any_::__receiver_ref<stdexec::completion_signatures<stdexec::__rcvrs::set_value_t(), stdexec::__rcvrs::set_error_t(std::__exception_ptr::exception_ptr), stdexec::__rcvrs::set_stopped_t()> >]'
   70 |   using connect_result_t = __call_result_t<connect_t, _Sender, _Receiver>;
      |         ^~~~~~~~~~~~~~~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__submit.hpp:131:11:   required from 'struct stdexec::submit_result<exec::__task::basic_task<void, exec::__task::__default_task_context_impl<exec::__task::__scheduler_affinity::__sticky> >, stdexec::__any_::__receiver_ref<stdexec::completion_signatures<stdexec::__rcvrs::set_value_t(), stdexec::__rcvrs::set_error_t(std::__exception_ptr::exception_ptr), stdexec::__rcvrs::set_stopped_t()> >, stdexec::__submit_result_kind::__connect>'
  131 |     using __result_t = connect_result_t<_Sender, _Receiver>;
      |           ^~~~~~~~~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__variant.hpp:79:66:   required from 'constexpr const std::size_t stdexec::__var::__variant<0, stdexec::__monostate, stdexec::submit_result<exec::__task::basic_task<void, exec::__task::__default_task_context_impl<exec::__task::__scheduler_affinity::__sticky> >, stdexec::__any_::__receiver_ref<stdexec::completion_signatures<stdexec::__rcvrs::set_value_t(), stdexec::__rcvrs::set_error_t(std::__exception_ptr::exception_ptr), stdexec::__rcvrs::set_stopped_t()>, stdexec::__env::env<> >, stdexec::__submit_result_kind::__connect> >::__max_size'
   79 |       static constexpr std::size_t __max_size = stdexec::__umax({sizeof(_Ts)...});
      |                                                                  ^~~~~~~~~~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__variant.hpp:80:21:   required from 'class stdexec::__var::__variant<0, stdexec::__monostate, stdexec::submit_result<exec::__task::basic_task<void, exec::__task::__default_task_context_impl<exec::__task::__scheduler_affinity::__sticky> >, stdexec::__any_::__receiver_ref<stdexec::completion_signatures<stdexec::__rcvrs::set_value_t(), stdexec::__rcvrs::set_error_t(std::__exception_ptr::exception_ptr), stdexec::__rcvrs::set_stopped_t()>, stdexec::__env::env<> >, stdexec::__submit_result_kind::__connect> >'
   80 |       static_assert(__max_size != 0);
      |                     ^~~~~~~~~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__let.hpp:426:24:   required from 'struct stdexec::__let::__let_state<stdexec::__any_::__receiver_ref<stdexec::completion_signatures<stdexec::__rcvrs::set_value_t(), stdexec::__rcvrs::set_error_t(std::__exception_ptr::exception_ptr), stdexec::__rcvrs::set_stopped_t()> >, main()::<lambda()>, stdexec::__rcvrs::set_value_t, stdexec::__let::__unknown_scheduler, stdexec::__tup::__tuple<0> >'
  426 |       __submit_variant __storage_{};
      |                        ^~~~~~~~~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__let.hpp:509:69:   [ skipping 60 instantiation contexts, use -ftemplate-backtrace-limit=0 to disable ]
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__meta.hpp:467:42:   required from 'static auto stdexec::__sigs::get_completion_signatures_t::__impl() [with _Sender = stdexec::{anonymous}::__sexpr<<lambda closure object>stdexec::<lambda()>()>; _Env = {}]'
  467 |   concept __mvalid = requires { typename __meval<_Tp, _Args...>; };
      |                                 ~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__senders.hpp:150:93:   required by substitution of 'template<class _Sender, class ... _Env>  requires  sizeof ... (_Env ...) <= 1 constexpr decltype (__impl<_Sender, _Env ...>()()) stdexec::__sigs::get_completion_signatures_t::operator()(_Sender&&, _Env&& ...) const [with _Sender = stdexec::{anonymous}::__sexpr<<lambda closure object>stdexec::<lambda()>()>; _Env = {}]'
  150 |         operator()(_Sender&&, _Env&&...) const noexcept -> decltype(__impl<_Sender, _Env...>()()) {
      |                                                                     ~~~~~~~~~~~~~~~~~~~~~~~~^~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__meta.hpp:886:53:   required by substitution of 'template<class _Fun, class ... _As> using stdexec::__call_result_t = decltype (__declval<_Fun>()((__declval<_As>)()...)) [with _Fun = stdexec::__sigs::get_completion_signatures_t; _As = {stdexec::{anonymous}::__sexpr<<lambda closure object>stdexec::<lambda()>()>}]'
  886 |   using __call_result_t = decltype(__declval<_Fun>()(__declval<_As>()...));
      |                                    ~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__execution_fwd.hpp:159:9:   required by substitution of 'template<class _Sender, class ... _Env> using stdexec::__completion_signatures_of_t = stdexec::__call_result_t<stdexec::__sigs::get_completion_signatures_t, _Sender, _Env ...> [with _Sender = stdexec::{anonymous}::__sexpr<<lambda closure object>stdexec::<lambda()>()>; _Env = {}]'
  159 |   using __completion_signatures_of_t =
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__meta.hpp:389:11:   required by substitution of 'template<template<class ...> class _Fn, class ... _Args> using stdexec::__i<true, true>::__g = _Fn<_Args ...> [with _Fn = stdexec::__completion_signatures_of_t; _Args = {stdexec::{anonymous}::__sexpr<<lambda closure object>stdexec::<lambda()>()>}]'
  389 |     using __g = _Fn<_Args...>;
      |           ^~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__meta.hpp:367:9:   required by substitution of 'template<template<class ...> class _Fn, class ... _Args> using stdexec::__meval = typename stdexec::__i<_Ok<_Args ...> >::__g<template<class ...> class _Fn, _Args ...> [with _Fn = stdexec::__completion_signatures_of_t; _Args = {stdexec::{anonymous}::__sexpr<<lambda closure object>stdexec::<lambda()>()>}]'
  367 |   using __meval = typename __i<_Ok<_Args...>>::template __g<_Fn, _Args...>;
      |         ^~~~~~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__meta.hpp:467:42:   required by substitution of 'template<template<class ...> class _Try, class _Catch> template<class ... _Args> using stdexec::__mtry_catch_q<_Try, _Catch>::__f = stdexec::__minvoke<stdexec::__if_c<__mvalid<_Try, _Args ...>, stdexec::__q<_Fn>, _Catch>, _Args ...> [with _Args = {stdexec::{anonymous}::__sexpr<<lambda closure object>stdexec::<lambda()>()>}; _Try = stdexec::__completion_signatures_of_t; _Catch = stdexec::__mconst<stdexec::_ERROR_<stdexec::__detail::__dependent_completions> >]'
  467 |   concept __mvalid = requires { typename __meval<_Tp, _Args...>; };
      |                                 ~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__starts_on.hpp:58:80:   required from 'auto [requires stdexec::__well_formed_sender<<placeholder>, >] stdexec::__starts_on_ns::starts_on_t::operator()(_Scheduler&&, _Sender&&) const [with _Scheduler = stdexec::inline_scheduler; _Sender = stdexec::{anonymous}::__sexpr<<lambda closure object>stdexec::<lambda()>()>]'
   58 |             static_cast<_Scheduler&&>(__sched), static_cast<_Sender&&>(__sndr)));
      |                                                                                ^
/Users/kyusic/dev/stdexec/test_exec.cpp:5:21:   required from here
    5 |   stdexec::starts_on(
      |   ~~~~~~~~~~~~~~~~~~^
    6 |       stdexec::inline_scheduler{},
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    7 |       stdexec::just() |
      |       ~~~~~~~~~~~~~~~~~
    8 |           stdexec::let_value([]() -> exec::task<void> { co_return; }));
      |           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__meta.hpp:886:53: error: no match for call to '(stdexec::__connect::connect_t) (exec::__task::basic_task<void, exec::__task::__default_task_context_impl<exec::__task::__scheduler_affinity::__sticky> >, stdexec::__any_::__receiver_ref<stdexec::completion_signatures<stdexec::__rcvrs::set_value_t(), stdexec::__rcvrs::set_error_t(std::__exception_ptr::exception_ptr), stdexec::__rcvrs::set_stopped_t()> >)'
  886 |   using __call_result_t = decltype(__declval<_Fun>()(__declval<_As>()...));
      |                                    ~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~
In file included from /Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__as_awaitable.hpp:25,
                 from /Users/kyusic/dev/stdexec/include/exec/../stdexec/execution.hpp:21:
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__senders.hpp:268:12: note: candidate: 'template<class _Sender, class _Receiver> stdexec::__call_result_t<decltype (__select_impl<_Sender, _Receiver>())> stdexec::__connect::connect_t::operator()(_Sender&&, _Receiver&&) const'
  268 |       auto operator()(_Sender&& __sndr, _Receiver&& __rcvr) const
      |            ^~~~~~~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__senders.hpp:268:12: note:   template argument deduction/substitution failed:
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__meta.hpp: In substitution of 'template<class _Fun, class ... _As> using stdexec::__call_result_t = decltype (__declval<_Fun>()((__declval<_As>)()...)) [with _Fun = stdexec::__connect::_NO_USABLE_CONNECT_CUSTOMIZATION_FOUND_; _As = {}]':
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__senders.hpp:268:12:   required by substitution of 'template<class _Sender, class _Receiver> stdexec::__call_result_t<decltype (__select_impl<_Sender, _Receiver>())> stdexec::__connect::connect_t::operator()(_Sender&&, _Receiver&&) const [with _Sender = exec::__task::basic_task<void, exec::__task::__default_task_context_impl<exec::__task::__scheduler_affinity::__sticky> >; _Receiver = stdexec::__any_::__receiver_ref<stdexec::completion_signatures<stdexec::__rcvrs::set_value_t(), stdexec::__rcvrs::set_error_t(std::__exception_ptr::exception_ptr), stdexec::__rcvrs::set_stopped_t()> >]'
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__meta.hpp:886:53:   required by substitution of 'template<class _Fun, class ... _As> using stdexec::__call_result_t = decltype (__declval<_Fun>()((__declval<_As>)()...)) [with _Fun = stdexec::__connect::connect_t; _As = {exec::__task::basic_task<void, exec::__task::__default_task_context_impl<exec::__task::__scheduler_affinity::__sticky> >, stdexec::__any_::__receiver_ref<stdexec::completion_signatures<stdexec::__rcvrs::set_value_t(), stdexec::__rcvrs::set_error_t(std::__exception_ptr::exception_ptr), stdexec::__rcvrs::set_stopped_t()>, stdexec::__env::env<> >}]'
  886 |   using __call_result_t = decltype(__declval<_Fun>()(__declval<_As>()...));
      |                                    ~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__senders_core.hpp:70:9:   required by substitution of 'template<class _Sender, class _Receiver> using stdexec::connect_result_t = stdexec::__call_result_t<stdexec::__connect::connect_t, _Sender, _Receiver> [with _Sender = exec::__task::basic_task<void, exec::__task::__default_task_context_impl<exec::__task::__scheduler_affinity::__sticky> >; _Receiver = stdexec::__any_::__receiver_ref<stdexec::completion_signatures<stdexec::__rcvrs::set_value_t(), stdexec::__rcvrs::set_error_t(std::__exception_ptr::exception_ptr), stdexec::__rcvrs::set_stopped_t()> >]'
   70 |   using connect_result_t = __call_result_t<connect_t, _Sender, _Receiver>;
      |         ^~~~~~~~~~~~~~~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__submit.hpp:131:11:   required from 'struct stdexec::submit_result<exec::__task::basic_task<void, exec::__task::__default_task_context_impl<exec::__task::__scheduler_affinity::__sticky> >, stdexec::__any_::__receiver_ref<stdexec::completion_signatures<stdexec::__rcvrs::set_value_t(), stdexec::__rcvrs::set_error_t(std::__exception_ptr::exception_ptr), stdexec::__rcvrs::set_stopped_t()> >, stdexec::__submit_result_kind::__connect>'
  131 |     using __result_t = connect_result_t<_Sender, _Receiver>;
      |           ^~~~~~~~~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__variant.hpp:79:66:   required from 'constexpr const std::size_t stdexec::__var::__variant<0, stdexec::__monostate, stdexec::submit_result<exec::__task::basic_task<void, exec::__task::__default_task_context_impl<exec::__task::__scheduler_affinity::__sticky> >, stdexec::__any_::__receiver_ref<stdexec::completion_signatures<stdexec::__rcvrs::set_value_t(), stdexec::__rcvrs::set_error_t(std::__exception_ptr::exception_ptr), stdexec::__rcvrs::set_stopped_t()>, stdexec::__env::env<> >, stdexec::__submit_result_kind::__connect> >::__max_size'
   79 |       static constexpr std::size_t __max_size = stdexec::__umax({sizeof(_Ts)...});
      |                                                                  ^~~~~~~~~~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__variant.hpp:80:21:   [ skipping 62 instantiation contexts, use -ftemplate-backtrace-limit=0 to disable ]
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__meta.hpp:467:42:   required from 'static auto stdexec::__sigs::get_completion_signatures_t::__impl() [with _Sender = stdexec::{anonymous}::__sexpr<<lambda closure object>stdexec::<lambda()>()>; _Env = {}]'
  467 |   concept __mvalid = requires { typename __meval<_Tp, _Args...>; };
      |                                 ~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__senders.hpp:150:93:   required by substitution of 'template<class _Sender, class ... _Env>  requires  sizeof ... (_Env ...) <= 1 constexpr decltype (__impl<_Sender, _Env ...>()()) stdexec::__sigs::get_completion_signatures_t::operator()(_Sender&&, _Env&& ...) const [with _Sender = stdexec::{anonymous}::__sexpr<<lambda closure object>stdexec::<lambda()>()>; _Env = {}]'
  150 |         operator()(_Sender&&, _Env&&...) const noexcept -> decltype(__impl<_Sender, _Env...>()()) {
      |                                                                     ~~~~~~~~~~~~~~~~~~~~~~~~^~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__meta.hpp:886:53:   required by substitution of 'template<class _Fun, class ... _As> using stdexec::__call_result_t = decltype (__declval<_Fun>()((__declval<_As>)()...)) [with _Fun = stdexec::__sigs::get_completion_signatures_t; _As = {stdexec::{anonymous}::__sexpr<<lambda closure object>stdexec::<lambda()>()>}]'
  886 |   using __call_result_t = decltype(__declval<_Fun>()(__declval<_As>()...));
      |                                    ~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__execution_fwd.hpp:159:9:   required by substitution of 'template<class _Sender, class ... _Env> using stdexec::__completion_signatures_of_t = stdexec::__call_result_t<stdexec::__sigs::get_completion_signatures_t, _Sender, _Env ...> [with _Sender = stdexec::{anonymous}::__sexpr<<lambda closure object>stdexec::<lambda()>()>; _Env = {}]'
  159 |   using __completion_signatures_of_t =
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__meta.hpp:389:11:   required by substitution of 'template<template<class ...> class _Fn, class ... _Args> using stdexec::__i<true, true>::__g = _Fn<_Args ...> [with _Fn = stdexec::__completion_signatures_of_t; _Args = {stdexec::{anonymous}::__sexpr<<lambda closure object>stdexec::<lambda()>()>}]'
  389 |     using __g = _Fn<_Args...>;
      |           ^~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__meta.hpp:367:9:   required by substitution of 'template<template<class ...> class _Fn, class ... _Args> using stdexec::__meval = typename stdexec::__i<_Ok<_Args ...> >::__g<template<class ...> class _Fn, _Args ...> [with _Fn = stdexec::__completion_signatures_of_t; _Args = {stdexec::{anonymous}::__sexpr<<lambda closure object>stdexec::<lambda()>()>}]'
  367 |   using __meval = typename __i<_Ok<_Args...>>::template __g<_Fn, _Args...>;
      |         ^~~~~~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__meta.hpp:467:42:   required by substitution of 'template<template<class ...> class _Try, class _Catch> template<class ... _Args> using stdexec::__mtry_catch_q<_Try, _Catch>::__f = stdexec::__minvoke<stdexec::__if_c<__mvalid<_Try, _Args ...>, stdexec::__q<_Fn>, _Catch>, _Args ...> [with _Args = {stdexec::{anonymous}::__sexpr<<lambda closure object>stdexec::<lambda()>()>}; _Try = stdexec::__completion_signatures_of_t; _Catch = stdexec::__mconst<stdexec::_ERROR_<stdexec::__detail::__dependent_completions> >]'
  467 |   concept __mvalid = requires { typename __meval<_Tp, _Args...>; };
      |                                 ~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__starts_on.hpp:58:80:   required from 'auto [requires stdexec::__well_formed_sender<<placeholder>, >] stdexec::__starts_on_ns::starts_on_t::operator()(_Scheduler&&, _Sender&&) const [with _Scheduler = stdexec::inline_scheduler; _Sender = stdexec::{anonymous}::__sexpr<<lambda closure object>stdexec::<lambda()>()>]'
   58 |             static_cast<_Scheduler&&>(__sched), static_cast<_Sender&&>(__sndr)));
      |                                                                                ^
/Users/kyusic/dev/stdexec/test_exec.cpp:5:21:   required from here
    5 |   stdexec::starts_on(
      |   ~~~~~~~~~~~~~~~~~~^
    6 |       stdexec::inline_scheduler{},
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    7 |       stdexec::just() |
      |       ~~~~~~~~~~~~~~~~~
    8 |           stdexec::let_value([]() -> exec::task<void> { co_return; }));
      |           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__meta.hpp:886:53: error: use of deleted function 'void stdexec::__connect::_NO_USABLE_CONNECT_CUSTOMIZATION_FOUND_::operator()() const'
  886 |   using __call_result_t = decltype(__declval<_Fun>()(__declval<_As>()...));
      |                                    ~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~
/Users/kyusic/dev/stdexec/include/exec/../stdexec/__detail/__senders.hpp:190:12: note: declared here
  190 |       void operator()() const noexcept = delete;
      |            ^~~~~~~~
```

### Other similar codes which succeed to compile

```c++
  stdexec::on(
      stdexec::inline_scheduler{},
      stdexec::just() |
          stdexec::let_value([]() -> exec::task<void> { co_return; }));
```

```c++
  stdexec::starts_on(stdexec::inline_scheduler{},
                     ([]() -> exec::task<void> { co_return; })());
```

### Environment

- OS: MacOS 15.6
- compiler: gcc version 14.2.0 (MacPorts gcc14 14.2.0_3+stdlib_flag)
- compiler flags: -g -std=gnu++20 -arch arm64 -fcoroutines -fconcepts-diagnostics-depth=10
- stdexec git commit: e5dbe49a5257d29adee456f9f653acb00df634f5

--- a/include/exec/task.hpp
+++ b/include/exec/task.hpp
@@ -494,22 +494,6 @@ namespace exec {
         return __task_awaitable<>{std::exchange(__coro_, {})};
       }
 
-      // From the list of types [_Ty], remove any types that are void, and send
-      //   the resulting list to __qf<set_value_t>, which uses the list of types
-      //   as arguments of a function type. In other words, set_value_t() if _Ty
-      //   is void, and set_value_t(_Ty) otherwise.
-      using __set_value_sig_t = __minvoke<__mremove<void, __qf<set_value_t>>, _Ty>;
-
-      // Specify basic_task's completion signatures
-      //   This is only necessary when basic_task is not generally awaitable
-      //   owing to constraints imposed by its _Context parameter.
-      using __task_traits_t =
-        completion_signatures<__set_value_sig_t, set_error_t(std::exception_ptr), set_stopped_t()>;
-
-      auto get_completion_signatures(__ignore = {}) const -> __task_traits_t {
-        return {};
-      }
-
       explicit basic_task(__coro::coroutine_handle<promise_type> __coro) noexcept
         : __coro_(__coro) {
       }

